backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -107,12 +115,70 @@ describe("Listings", () => {
           zipcode: "94621,94404",
         },
       ],
+      view: "base",
     }
     const query = qs.stringify(queryParams)
     const res = await supertest(app.getHttpServer()).get(`/listings?${query}`).expect(200)
     expect(res.body.items.length).toBeGreaterThanOrEqual(2)
   })
 
+  it("should return listings with matching Alameda jurisdiction", async () => {
+    const jurisdictions = await jurisdictionsRepository.find()
+    expect(jurisdictions.length).toBe(4)
```

Comment:
I don't think this check brings anything to the test other than breaking a test suite when we add a new jurisidiction to the seeds.

Reply:
Same comment for other tests that check the length.